src/components/MapCanvas.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+
+import summary from '../apis/summary';
+class MapCanvas extends Component {
```

Comment:
I should introduce an ESLint rule here, but we usually want spacing between the imports and the code blocks they support!

Reply:
yes, if you have time. This will make it more consistent